docs/sources/configuration/index.md

```diff
@@ -58,7 +58,7 @@ Direct DB Connection allows plugin to use existing SQL data source for querying
 database. This way usually faster than pulling data from Zabbix API, especially on the wide time ranges, and reduces
 amount of data transferred.
 
-Read [how to configure](./sql_datasource) SQL data source in Grafana.
+Read [how to configure](./direct_db_datasource.md) SQL data source in Grafana.
```

Comment:
You don't need to append link with `.md`
```suggestion
Read [how to configure](./direct_db_datasource) SQL data source in Grafana.
```

Reply:
right, remove it~